net-libs/libwebsockets/libwebsockets-1.7.7.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="canonical libwebsockets.org websocket library"
+HOMEPAGE="https://libwebsockets.org/"
+SRC_URI="https://github.com/warmcat/libwebsockets/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+ssl +zlib static-libs +http2 ipv6 +server client libressl libev libuv"
+
+RDEPEND="
+	ssl? (
+		!libressl? ( dev-libs/openssl:0=  )
+		libressl?  ( dev-libs/libressl:0=  )
+		)
+	ssl?           ( dev-libs/openssl:0 )
```

Comment:
Same here, you should remove this USE dependency line because you have one for libressl